modules/nlu/src/views/full/entities/EntityEditor.jsx

```diff
@@ -109,6 +89,12 @@ export default class EntityEditor extends React.Component {
     this.setState({ currentEntity: entity, currentOccurence: entity.occurences[0] }, this.onUpdate)
   }
 
+  handleTagChange = occurenceIndex => synonyms => {
```

Comment:
thanks for those changes, I'll pick them straight up for my ongoing rewriter of this UI